plutus-core/plutus-core/src/PlutusCore/TypeCheck.hs

```diff
@@ -36,9 +36,9 @@ import PlutusCore.Quote
 import PlutusCore.Rename
 import PlutusCore.TypeCheck.Internal
 
--- | The constraint for built-in types/functions are kind/type-checkable.
+-- | The constraint for built-in types\/functions are kind\/type-checkable.
```

Comment:
Here's how the previous version was rendered:

![Screenshot from 2024-05-18 09-59-35](https://github.com/IntersectMBO/plutus/assets/10480926/020f6212-984c-4506-956a-bd3f0dd134d7)